savoirfairelinux/ringme.js
# build: add eslint and linting integration

Fixes #30

## Adds linting via ESLint

### Changes proposed in this pull request:

- Adds linting!
- uses eslint-config-standard (but other config--e.g. airbnb, exist as well!)
- Fixes any linting errors

### Status


### How to verify this change

- Check out project
- run `yarn` or `npm install`
- Run `yarn lint` or `npm run lint` to validate that linting is running, and that linting passes

### Additional notes

- eslint complained about an unnecessary .call (L147-149), so I switched to bind; that said, I don't think you actually need it, but didn't want to break functionality!
- I turned off a rule `brace-style`, but may want to actually look into fixing it at some point
- Some tests would be awesome too :)
